apps/src/applab/constants.js

```diff
@@ -1,5 +1,9 @@
 module.exports = {
   FOOTER_HEIGHT: 30,
   DESIGN_ELEMENT_ID_PREFIX: 'design_',
-  NEW_SCREEN: "New screen..."
+  NEW_SCREEN: "New screen...",
+  MODE: {
+    CODE: 'CODE',
+    DESIGN: 'DESIGN'
```

Comment:
:frowning: Yes, this one is, but it's also an ID that we depend on to know that we're creating a new screen.

```javascript
if (screenId === constants.NEW_SCREEN) {
  screenId = this.props.onScreenCreate();
}
```
We could make this check the loc'd string, but I'd rather change this to check something else.  I don't think we prevent the user from making a screen called "New screen..."...